lib/core/services/alfresco-api.service.ts

```diff
@@ -35,7 +35,7 @@ export class AlfrescoApiService {
 
     alfrescoApiInitialized: ReplaySubject<boolean> = new ReplaySubject(1);
 
-    protected alfrescoApi: AlfrescoApi;
+    protected alfrescoApi: AlfrescoApi = new AlfrescoApi();
```

Comment:
hmm, I see you like living the dangerous way :trollface: 
Why not in the constructor?

Reply:
I Need a reference at the constructor time if it get called after the affects I will have null in the instance I got there.